exercises/transpose/transpose.test.ts

```diff
@@ -120,7 +120,29 @@ describe("Transpose", () => {
       "     R"
     ]
     expect(
-      Transpose.transpose(["T", "EE", "AAA", "SSSS", "EEEEE", "RRRRRR"])
+      transpose(["T", "EE", "AAA", "SSSS", "EEEEE", "RRRRRR"])
+    ).toEqual(expected)
+  })
+
+  xit("jagged triangle", () => {
```

Comment:
Only request I have is to also push this addition to `problem-specifications`.

1. generate a uuid (google a generator, generate one, or use `configlet uuid` if you have `configlet`)
2. add it to [`.meta/tests.toml`](https://github.com/exercism/typescript/blob/master/exercises/transpose/.meta/tests.toml)
3. PR it to [_this file_](https://github.com/exercism/problem-specifications/blob/master/exercises/transpose/canonical-data.json#L218-L240). 

I'm also in that repo, the process won't be lengthy to get it merged there.

Reply:
A.) problem-specification pull request: https://github.com/exercism/problem-specifications/pull/1748
In hindsight you specified a range - did you want to replace the "triangle" test with "jagged triangle"? 
Because I simply added a new test at the end.

B.) added the uuid to the tests.toml

C.) "package.json are changed from 100644 → 100755"?
It's something that must have happened when I ran through the lint, test, sync, ci-check, ci scripts.

Reply:
C) Yeah it was possibly wrong already 💯 . I fixed it :)

Reply:
A) as you've done it right now looks perfect to me.